lib/src/query_selector.dart

```diff
@@ -40,21 +40,19 @@ class SelectorEvaluator extends Visitor {
   }
 
   Element querySelector(Node root, SelectorGroup selector) {
-    for (var node in root.nodes) {
-      if (node is! Element) continue;
-      if (matches(node, selector)) return node;
-      var result = querySelector(node, selector);
+    for (var element in root.nodes.whereType<Element>()) {
```

Comment:
Probably doesn't matter but the previous implementation could have been done for speed reasons?

Reply:
When it was first written we didn't have `whereType` as an option. I don't _think_ the performance is going to matter much here, so I preferred to use this over introducing yet another local variable or adding multiple casts.